helpers/validatelockcreation.js

```diff
@@ -0,0 +1,111 @@
+function validateCommonInputs(inputs, validationErrors) {
+  // rearrange in order of CreatedLockFields to make sure everything gets validated
+  if(inputs.Shared != false && inputs.Shared != true) {
+    validationErrors.push("Shared is not valid");
+  }	// enforced by schema?
```

Comment:
I thought so.  The same applies for a bunch of other checks where the schema says Boolean!    It might be best to leave these in though, for a while in case we mess up the schema by making changes to it.